python/bolt-mcp/tests/integration/test_mcp_oauth_integration.py

```
@@ -0,0 +1,268 @@
+_CREATE_USER = (
+    "from django.contrib.auth import get_user_model as M;"
+    f"u = M().objects.create_user({USERNAME!r}, password={PASSWORD!r});"
+    "u.is_staff = True; u.save()"
+)
+
+
+def _manage(project, *args: str) -> None:
+    """Run a manage.py command in the generated project (same interpreter + PYTHONPATH)."""
+    env = os.environ.copy()
+    env["PYTHONPATH"] = os.pathsep.join(filter(None, [str(project.root), env.get("PYTHONPATH", "")]))
+    result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)